main/autogen/subclasses.cc

```diff
@@ -113,13 +113,21 @@ vector<string> Subclasses::serializeSubclassMap(const Subclasses::Map &descendan
         auto type = children.classKind == ClassKind::Class ? "class" : "module";
         descendantsMapSerialized.emplace_back(fmt::format("{} {}", type, parentName));
 
+        const auto classFormatString = showPaths ? " class {} {}" : " class {}";
+        const auto moduleFormatString = showPaths ? " module {} {}" : " module {}";
```

Comment:
Let's hoist these out of the loop since they seem the same regardless of `parentName`.